### PR TITLE
Build Carolina

### PR DESCRIPTION
**Pull request by pgdr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,12 @@ install:
   - cd dakota-6.6.0.src/
   - mkdir build
   - cd build
-  - cmake -DDAKOTA_HAVE_MPI=FALSE ..
+  - export DAKOTA_INSTALL=$PWD/install
+  - cmake -DDAKOTA_HAVE_MPI=FALSE -DCMAKE_INSTALL_PREFIX=$DAKOTA_INSTALL ..
   - make -j 3
```
Comment (joakim-hove) [inline .travis.yml]: Missing `make install`?

**Comment by pgdr** [comment]
I fear that this is not going to go through due to exceeding Travis memory limits.

I suggest that we merge this and awaits the deb package we are currently building.

```make
...
[ 82%] Building CXX object src/CMakeFiles/dakota_library_mode.dir/library_mode.cpp.o
Building CXX object src/CMakeFiles/dakota.dir/main.cpp.o
Linking CXX executable dakota
Linking CXX executable dakota_restart_util
Linking CXX executable dakota_library_mode
collect2: error: ld terminated with signal 9 [Killed]
make[2]: *** [src/dakota_restart_util] Error 1
make[1]: *** [src/CMakeFiles/dakota_restart_util.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 82%] Built target dakota
[ 82%] Built target dakota_library_mode
make: *** [all] Error 2
The command "make -j 3 install" failed and exited with 2 during .
Your build has been stopped.
```

**Comment by joakim-hove** [comment]
Do you actually mean *memory* limits - or time limits?

**Comment by pgdr** [comment]
Memory limits.  The build gets killed by Travis after ~20 minutes.

**Pull request merged**